pangeo/values.yaml

```diff
@@ -47,3 +47,22 @@ jupyterhub:
 #    memory:
 #      limit: 4G
 #      guarantee: 2G
+
+gateway:
```

Comment:
I think this also needs to be under a `dask-gateway` key, but maybe not. I'm not very familiar with chart dependencies.

Reply:
Yeah it should be:
```yaml
dask-gateway:
  gateway:
    # Everything else here
```